.devcontainer/devcontainer.json

```diff
@@ -1,26 +1,25 @@
 {
     "name": "Azure Developer CLI",
-    "image": "mcr.microsoft.com/devcontainers/javascript-node:18-bookworm",
+    "image": "mcr.microsoft.com/devcontainers/typescript-node:dev-18-bookworm",
     "features": {
-        "ghcr.io/azure/azure-dev/azd:latest": {},
-        "ghcr.io/devcontainers/features/azure-cli:latest": {},
         "ghcr.io/devcontainers/features/docker-in-docker:latest": {},
-        "ghcr.io/devcontainers/features/github-cli:latest": {},
-        "ghcr.io/devcontainers/features/powershell:latest": {}
```

Comment:
why remove Powershell?

Reply:
@dotnet-7 It used to be there in my original `devcontainer.json` settings. Please revert.